### DISCUSSION TITLE
How to jump back to a running Airflow Docker Container? When ```docker exec -it container_id bash``` is not the issue

### DISCUSSION THREAD

**Discussion by stromal** [apache/airflow#26657] [Q&A]
- If I run the standard Airflow Docker [LINK](https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html)

- as ```docker-compose up -d```  run my instance I do not know where can I find this specific container ID. 

- I look up the running container IDs with ```docker ps -a```. I dont have any other containers or images just this project.

- I have entered in to all 7 container ```docker exec -it container_id bash``` I can enter every container no problem.
- But I don't get the same window and texts as if I would have stated it with only ```docker-compose up``` and that is where I want to get

- So how can I jump back to a running Airflow Docker Container when it is already running?

**Comment by potiuk** [comment]
there is nothing airflow specific there. This is the same as any other docker compose. The things that are displayed when you start are not 'container" they are logs printed by docker compose and you can access them with "logs" commands (assuming I understood what you asked for - hard to say as it was not clear what you want to do).

I recommend to go through Docker compose tutorial https://docs.docker.com/compose/gettingstarted/ - there are also video tutorials available. Again - Airflow is no different than other docker-compose app and you need to learn Docker Compose first if you want to manage Airflow with it.
